build.gradle.kts

```diff
@@ -28,3 +28,8 @@ subprojects {
 tasks.register("clean", Delete::class) {
     delete(rootProject.buildDir)
 }
+buildscript {
```

Comment:
Is there a reason you declared the Kotlin plugin again

Reply:
That's most likely a mistake of some sort. I can't remember adding it at all